app/views/well_known/keybase_proof_config/show.json.ruby

```diff
@@ -0,0 +1,32 @@
+# see https://keybase.io/docs/proof_integration_guide#1-config
+
+JSON.pretty_generate({
```

Comment:
To generate JSON, we use ActiveModelSerializers, see e.g. `app/serializers/manifest_serializer.rb`

Reply:
yeah. i had an issue with json parsing URLs with multiple query string params (`&` -> `/u0026`). but whatever. i fixed it on the keybase side, and now this looks better here.